config/local.conf

```diff
@@ -1,6 +1,8 @@
 <?xml version="1.0"?>
 <!DOCTYPE fontconfig SYSTEM "fonts.dtd">
 <fontconfig>
+    <!-- Add shared WSLg Font directory -->
+    <dir>/mnt/wslg/fonts</dir>
```

Comment:
I believe this only add font path to fontconfig backend (like fc-list), but not to X server side font (such as xlsfonts). For X server fonts, "FontPath" needs to be added by xorg.conf as well, I think.